test/fixtures/long-running/jobs.yml.erb

```diff
@@ -0,0 +1,20 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: jobs
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        name: jobs
+        app: fixtures
+    spec:
+      containers:
+      - name: app
+        image: kirshatrov/sleep-forever:latest
```

Comment:
This is https://github.com/kirs/docker-run-forever/blob/master/sleep.rb packed into Docker image.